histolab/masks.py

```diff
@@ -197,7 +197,4 @@ def _(self, tile: Tile) -> np.ndarray:
         mask: np.ndarray
```

Comment:
docstring.

Is it necessary to have this as a method?

Reply:
> docstring. 

what do you mean?

> Is it necessary to have this as a method?

I like the idea of having `TissueMask` that is flexible and does the right thing based on the input

@ernestoarbitrio thoughts?

Reply:
I do not understand the matter ... sorry